js/main.js

```diff
@@ -47,41 +80,63 @@ function renderEntry(entry) {
   var description = document.createElement('p');
   description.textContent = entry.notes;
   var image = document.createElement('img');
-  image.setAttribute('src', entry.photoUrl);
+  image.setAttribute('src', entry.photo);
   var heading = document.createElement('h3');
   heading.textContent = entry.title;
-
+  var editIcon = document.createElement('i');
+  editIcon.className = 'fas fa-pen';
   list.appendChild(firstDiv);
   firstDiv.appendChild(colHalfdiv);
   colHalfdiv.appendChild(image);
   firstDiv.appendChild(secondcolHalf);
   secondcolHalf.appendChild(heading);
   secondcolHalf.appendChild(description);
-  var ul = document.querySelector('.parent');
-  ul.prepend(list);
-
+  heading.appendChild(editIcon);
   return list;
 }
-window.addEventListener('DOMContentLoaded', DOMContentLoaded);
 
+function editClick(event) {
+  // var list = document.querySelector('li');
+  data.view = 'entry-form';
+  var toEdit = event.target.closest('li');
+  var entryId = toEdit.getAttribute('data-entry-id');
+  var entry = data.entries.find(entry => entry.id == (entryId));
+  if (event.target.tagName === 'I') {
+    EntriesText.textContent = 'Edit Entry';
+    $EntriesView.className = 'hidden';
+    $entryForm.className = '';
+    data.editing = entry;
+    var $title = document.querySelector('#title');
+    var $notes = document.querySelector('#notes');
+    var $photoUrl = document.querySelector('#photoUrl');
+    var existingEntryId = document.querySelector('#existingEntryId');
+    $title.value = (data.editing.title);
+    $notes.value = (data.editing.notes);
+    $photoUrl.value = (data.editing.photo);
+    img.setAttribute('src', $photoUrl.value);
+    existingEntryId.value = (data.editing.id);
+  }
+}
+window.addEventListener('DOMContentLoaded', DOMContentLoaded);
 function DOMContentLoaded(event) {
```

Comment:
This function should be lowercase, only Classes in JS are typically capitalized.